splunkoncall.json

```diff
@@ -649,7 +649,7 @@
                 {
                     "data_path": "action_result.summary.entity_id",
                     "contains": [
-                        "victorops id"
+                        "splunk on call id"
```

Comment:
How does changing the contains affect contextual actions? ie could it make sense to keep both `victorops id` in addition to `splunk on call id`?

Reply:
Actually, we have replaced the 'victorops' with 'splunk on call' in the entire code. We have also verified the contextual actions where we have made changes in contains and they are working fine.